runner_internal_test.go

```diff
@@ -81,20 +167,14 @@ func TestRunnerGroupTaskAndFlagArgs(t *testing.T) {
 				"mock/task/3": {"-d=1"},
 			},
 		},
-		{
-			description: "Should exclude flags and flag args passed directly to taskrunner",
-			cliArgs:     []string{"--config", "pathtoconfig", "--listAll", "--watch", "mock/task/1", "--longFlagA"},
-			expectedGroups: map[string][]string{
-				"mock/task/1": {"--longFlagA"},
-			},
-		},
```

Comment:
This test case is no longer relevant due to the change on line 205 (post-change) of `runner.go`.